nomenclature/testing.py

```diff
@@ -27,20 +27,40 @@ def assert_valid_yaml(path: Path):
         )
 
 
-def assert_valid_structure(path: Path):
+def assert_valid_structure(
+    path: Path, mappings: str = "mappings", definitions: str = "definitions"
+) -> None:
     """Assert that `path` can be initialized as a :class:`DataStructureDefinition`
+    with optionnaly non-default folder names
```

Comment:
There is a typo in this line, but it's anyway better practice to keep the first sentence of a docstring to one line. And optional stuff doesn't need to be mentioned in the first line.
```suggestion
```